packages/graphql_server/src/api/openAIApi.ts

```diff
@@ -60,6 +60,10 @@ async function getELI5FromReadMe(readMe: string) {
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 async function getHackernewsSentiment(comments: string) {
+  // limit length of name to 5000 characters, because it might cause problems with GPT otherwise
+  if (comments.length > 5000) {
```

Comment:
`5000` should probably be a `const` called `GPT_MESSAGE_LIMIT` or something instead of a magic number :)